otx/algorithms/action/configs/classification/x3d/template.yaml

```diff
@@ -0,0 +1,64 @@
+# Description.
+model_template_id: Custom_Action_Classificaiton_X3D
+name: X3D
+task_type: ACTION_CLASSIFICATION
+task_family: VISION
+instantiation: "CLASS"
+summary: Basic transfer learning template for X3D
+application: ~
+
+# Algo backend.
+framework: OTEDetection v2.9.1
```

Comment:
There is no other way to get into Geti right now, so you can change the name to something like OTXAction for now.

Reply:
Done